packages/doc/docs/data-privacy/authn.mdx

```diff
@@ -15,8 +16,163 @@ VulcanSQL offers a range of built-in authenticators, each with its unique method
 
 Here is a list of built-in authenticators available in VulcanSQL:
 
-* **HTTP Basic**: Authenticate users via HTTP basic authentication. This method requires users to provide a username and password, which are transmitted as headers in each request.
-* **Password File**: Validate users' credentials using a password file. This method involves comparing the provided username and password against a pre-defined list of authorized users stored in a file.
-* **Simple Token**: Authenticate users with a static token. This method involves checking the provided token against a pre-configured token value.
+* [**HTTP Basic**](#http-basic): Authenticate users via HTTP basic authentication. This method requires users to provide a username and password, which are transmitted as headers in each request.
+* [**Password File**](#password-file): Validate users' credentials using a password file. This method involves comparing the provided username and password against a pre-defined list of authorized users stored in a file.
+* [**Simple Token**](#simple-token): Authenticate users with a static token. This method involves checking the provided token against a pre-configured token value.
 
 By implementing the appropriate authenticators for your VulcanSQL application, you can ensure that only authorized users access your system and protect sensitive data from unauthorized access.
+
+## HTTP Basic
+
+With HTTP basic authentication, VulcanSQL authenticates users via [HTTP Basic Authentication](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication).
+
+### Configuration
+
+Please fill out the values in your `vulcan.yaml`.
+
+```yaml
+auth:
+  enabled: true
+  options:
+    basic:
+      # Read users and passwords from a text file.
+      htpasswd-file:
+        path: passwd.txt # Path to the password file.
+        users: # (Optional) Add attributes for users
+          - name: eason
+            attr:
+              department: engineer
+```
+
+### Password file
```

Comment:
How about also adding the `user-list` section to complete the setting method? See: https://vulcan-sql-document-git-chore-bump-to-040-vulcan-sql-document.vercel.app/docs/api-building/access-control/authenticators/http-basic#user-list

Reply:
After a discussion with @cyyeh, we won't add the content to the document.